CHANGELOG.md

```diff
@@ -12,17 +12,7 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 
 ### Changed
 
-- Nodes will assume that all PoX anchor blocks exist by default, and stall initial block download indefinitely to await their arrival (#5502)
```

Comment:
Did we lose this bullet in the merge?

Reply:
definitely should not be removed. i'll double-check

Reply:
[f97a6be](https://github.com/stacks-network/stacks-core/pull/5607/commits/f97a6bea4d15901b59521331791b936c16fdb72e)